transport/grpc/dial.go

```diff
@@ -135,12 +134,11 @@ func dial(ctx context.Context, insecure bool, o *internal.DialSettings) (*grpc.C
 		}
 
 		// Attempt Direct Path only if:
+    // * Yoshi library set AttemptDirectPath option
```

Comment:
nit: remove

Let's not have anything any internal names mentioned in code.

Reply:
Done.